modules/olm-mirroring-catalog.adoc

```diff
@@ -0,0 +1,300 @@
+// Module included in the following assemblies:
+//
+// * operators/olm-restricted-networks.adoc
+// * operators/olm-managing-custom-catalogs.adoc
+// * migration/migrating_3_4/deploying-cam-3-4.adoc
+// * migration/migrating_4_1_4/deploying-cam-4-1-4.adoc
+// * migration/migrating_4_2_4/deploying-cam-4-2-4.adoc
+
+[id="olm-mirror-catalog_{context}"]
+= Mirroring an Operator catalog
+
+You can mirror the Operator content of a Red Hat-provided catalog, or a custom
+catalog, into a container image registry using the `oc adm catalog mirror`
+command. The target registry must support
+link:https://docs.docker.com/registry/spec/manifest-v2-2/[Docker v2-2]. For a
+cluster on a restricted network, this registry can be a registry that the
+cluster has network access to, such as a mirror registry created during a
+restricted network cluster installation.
+
+[IMPORTANT]
+====
+The internal registry of your {product-title} cluster cannot be used as the
```

Comment:
the mirror step now generates tags for any images that lack them (for a different reason - so that registries that GC untagged images don't GC the mirrored content). I haven't tried in a while but I'm wondering if mirroring to the internal registry works now

Reply:
I tried this out and it seems like it mostly works (was able to use the new-to-me `oc registry login` even to login, quite handy), but when testing out installing Operators from the resulting catalog, I wasn't 100% sure if they were coming from the right place or not (since my test cluster was not disconnected). Probably need more testing on this, but can update in a follow-up PR.

Reply:
I'll at least remove the warning here now since the reasoning is no longer accurate.